backend/videoPortal/videoPortal/Controllers/PlaylistController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading.Tasks;
+using videoPortal.DbContext;
+using videoPortal.Models;
+
+namespace videoPortal.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class PlaylistController : Controller
+    {
+        videoPortalDbContext dbContext;
+
+        public PlaylistController(videoPortalDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Get()
+        {
+            return Ok(await dbContext.Playlists.ToListAsync());
+        }
+
+
+        [HttpGet]
+        [Route("{id:guid}")]
+        public async Task<IActionResult> GetPlaylist([FromRoute] string id)
+        {
+            var pl = await dbContext.Playlists.FindAsync(id);
+
+            if (pl == null)
+            {
+                return NotFound();
+            }
+
+            return Ok(pl);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreatePlaylist(Playlist playlist)
+        {
+            var v = new Playlist()
+            {
+                Id = Guid.NewGuid().ToString(),
+                Title = playlist.Title,
+                Creator = playlist.Creator,
+                Playtime = playlist.Playtime
+            };
+
+            await dbContext.Playlists.AddAsync(v);
```

Comment:
try catch becuase if db drop connection or something make fail the addAsync or savecontex the app will die.